driver-core/src/test/resources/command-monitoring/command.json

```diff
@@ -1,17 +1,13 @@
 {
   "data": [
-    {
-      "_id": 1,
-      "x": 11
-    }
   ],
   "collection_name": "test",
   "database_name": "command-monitoring-tests",
   "tests": [
     {
       "description": "A successful command",
       "operation": {
-        "name": "count",
+        "name": "find",
```

Comment:
Changing spec tests to not use deprecated count anymore.  If they look good here I will make a PR against the specs repo.

Reply:
ACK